iceoryx_examples/icedocker/README.md

```diff
@@ -0,0 +1,105 @@
+# Use Iceoryx In A Docker Environment
+
+## Introduction
+
+Let's assume we are working on a system in which `iox-roudi` runs in a docker
+environment and it should orchestrate two applications which are running again
+in two different docker containers so that we end up with a system of 3
+different docker containers.
+
+To demonstrate the setup we use the 
+[icedelivery C++ example](https://github.com/eclipse-iceoryx/iceoryx/tree/master/iceoryx_examples/icedelivery).
+
+```
+                         +-----------+
+                         | docker 1  |
+                         |           |
+                         | iox-roudi |
+                         +-----------+
+
+       +-------------------+        +--------------------+
+       | docker 2          | send   | docker 3           |
+       |                   |------->|                    |
+       | iox-cpp-publisher | data   | iox-cpp-subscriber |
+       +-------------------+        +--------------------+
+```
+
+## Requirements
+
+### Shared Access to Unix Domain Sockets
+
+Every iceoryx application is registering itself at our central broker RouDi
+by sending a message to the unix domain socket located at
+`IOX_UDS_SOCKET_PATH_PREFIX/roudi` which is defined in the corresponding
+platform settings file `platform_settings.hpp`. In linux the socket file handle
+can be found at `/tmp/roudi`. While registering it announces its unix
```

Comment:
Is it possible that multiple docker container have the share memory from the host system but a local `tmp` directory, i.e. a local lock file? This would result in multiple RouDis interfering with each other

Reply:
Yes it is possible and then there are elephants with red eyes hiding in cherry trees

Reply:
Funnily it doesn't even crash the running configuration. It will keep running until that RouDi is stopped and new apps will use the new RouDi